build/main.js

```diff
@@ -53,7 +53,6 @@ const handleDepsCheckSucceeded = () => {
   const argv = process.argv;
   const tasks = require('./tasks').default;
   const handlers = {
-    '--build': args => tasks.build(args),
```

Comment:
We use this for our postinstall step so after download/install users mostly have everything pre-compiled already. Not against dropping that if that is complex to do now though.